src/sql/azdata.proposed.d.ts

```diff
@@ -949,6 +949,12 @@ declare module 'azdata' {
 		 * @param workspacefile
 		 */
 		export function enterWorkspace(workspaceFile: vscode.Uri): Promise<void>;
+
+		/**
+		 * Saves and enters the workspace with the provided path
+		 * @param workspacefile
+		 */
+		export function saveWorkspace(workspaceFile: vscode.Uri): Promise<void>;
```

Comment:
Shouldn't this be `saveAndEnterWorkspace` then?

Reply:
updated this one and also createWorkspace